src/components/atoms/Button/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import ButtonIcon from '../Button-icon'
+
+import styles from './styles.css'
+
+const Button = ({ onClick, text }) => (
```

Comment:
what is `text`? 
which occurs when I don't wanna an icon here?
can you add classNames as props?

Reply:
* We removed text and now the button title is children.
* About the icon we added a statement to check the iconURL prop.
* classNames added ✅